UnityProject/Assets/Scripts/Factories/EffectsFactory.cs

```diff
@@ -96,8 +96,12 @@ public static void BloodSplat(Vector3 worldPos, BloodSplatSize splatSize, BloodS
 
 		if (chosenTile != null)
 		{
-			Spawn.ServerPrefab(chosenTile, worldPos,
-				MatrixManager.AtPoint(Vector3Int.RoundToInt(worldPos), true).Objects);
+			var matrix = MatrixManager.AtPoint(Vector3Int.RoundToInt(worldPos), true).Matrix;
```

Comment:
don't call AtPoint twice, just cache `MatrixManager.AtPoint(Vector3Int.RoundToInt(worldPos), true)` here once:
```suggestion
			var matrixInfo = MatrixManager.AtPoint(Vector3Int.RoundToInt(worldPos), true);
```